docs/using.rst

```diff
@@ -804,6 +815,12 @@ By default, the following locations are searched:
   ``~/.config/letsencrypt/cli.ini`` if ``$XDG_CONFIG_HOME`` is not
   set).
 
+Since this configuration file applies to all invocations of certbot it is incorrect
+to list domains in it. Listing domains in cli.ini will prevent renewal from working.
+Additionally due to how arguments in cli.ini are parsed, options which wish to
+not be set should not be listed. Options set to ``= False`` will instead be read
+as being set to true, since they have been listed in the config file.
```

Comment:
This is no longer accurate with `configargparse 0.12.0`. #4995 aims to moves to pin this version in `certbot-auto`. Perhaps you should update this sentence to say:

"...read as being set to true by older versions of Certbot"

or

"...read as being set to true by older versions of Certbot's depdencies"